
# Name of the joint group
string groupName

# Interpolation space
uint8 interpolationSpace

# If interpolationSpace == IS_CARTESIAN, this is the target endeffector pose
geometry_msgs/Pose state
